### PR TITLE
Bugfix #89 fix clean-arlington

### PR DESCRIPTION
**Pull request by sstrochak**
Fill in lot size and number of units earlier in the process. Rerun vacant lots.

### CODE REVIEW

**Comment by sstrochak** [comment]
@lhendey do you need to review this? 
Changes to `clean-arlington.Rmd` are adding the spatial join and the merge with the address file after the collapse.
`preclean-arlington.Rmd` wan't actually changed, I just updated the comments to note that those two elements had to be added post-collapse.
I also reran the vacant lots and special exceptions vacant lots to account for the filled in lot size.

File: Prog/clean-arlington.Rmd [hunk 1]
```diff
@@ -1052,9 +1053,121 @@ rm(mult, other, nunits, cdties)
...
+rdir <- paste0("L:/Libraries/RegHsg/Raw/", filepath, "/")
+adfile <- paste0(rdir, filepath, "-address-file.csv")
+address <- read_csv(adfile)
+
+address1 <- address %>% 
+  mutate(address_cap = toupper(FULL_ADDRESS)) %>% 
+  select(address_cap, units_address = UNITCOUNT)
+
+jur5 <- jur4 %>% 
+  mutate(address_cap = toupper(propaddress)) %>%
```
Comment (lhendey) [inline Prog/clean-arlington.Rmd]: adding obs from jur4 to jur5, okay? 

Also getting:
Duplicated column names deduplicated: '-' => '-_1' [5], 'STATUS' => 'STATUS_1' [21]Parsed with column specification:
cols(
  .default = col_character(),
  `-` = col_logical(),
  Latitude = col_double(),
  Longitude = col_double(),
  Altitude = col_double(),
  `-_1` = col_logical(),
  OBJECTID = col_double(),
  STNUM = col_double(),
  UNITCOUNT = col_double(),
  UPDATE_DATE = col_datetime(format = ""),
  ZIP5 = col_double(),
  STATUS_1 = col_double()
)
See spec(...) for full column specifications.

File: Prog/clean-arlington.Rmd [hunk 2]
```diff
@@ -1052,9 +1053,121 @@ rm(mult, other, nunits, cdties)
...
+
+# convert arlington to spatial
+jursf <- jur5 %>% 
+  filter(!is.na(lat), !is.na(long)) %>% 
+  mutate(longx = long, laty = lat) %>% 
+  st_as_sf(coords = c("longx", "laty")) %>% 
+  st_set_crs(st_crs(4326))
+
+# join to zoning layer
+jurj <- st_join(jursf, par, st_intersects)
```
Comment (lhendey) [inline Prog/clean-arlington.Rmd]: do i need to be concerned about this one? 

"although coordinates are longitude/latitude, st_intersects assumes that they are planar"

**Review by lhendey — APPROVED** [review]
@sstrochak two comments

File: Prog/clean-arlington.Rmd (hunk 2 above)
Comment (sstrochak) [reply Prog/clean-arlington.Rmd]: @lhendey no :)

File: Prog/clean-arlington.Rmd (hunk 1 above)
Comment (sstrochak) [reply Prog/clean-arlington.Rmd]: this is just weird column names in the address file, didn't fix it since we only used two columns

Comment (sstrochak) [reply Prog/clean-arlington.Rmd]: looking into the increase in observations now

Comment (sstrochak) [reply Prog/clean-arlington.Rmd]: there were some observations in the address file that didn't have unique addresses. I collapsed them and summed the units (a lot of them had a value for one of the observations and 0 units for the remaining)

**Comment by lhendey** [comment]
@sstrochak is this okay to merge now?

**Comment by sstrochak** [comment]
@lhendey yeah, unless you want to go ahead and change the collapse?

**Pull request merged**